examples/message_decode/message_decode.cc

```diff
@@ -13,8 +13,12 @@ using namespace point_one::fusion_engine::messages;
 
 /******************************************************************************/
 bool DecodeMessage(std::ifstream& stream, size_t available_bytes) {
+  // Enforce a 4-byte aligned address.
   uint8_t storage[4096];
```

Comment:
can we just use alignas(4) on this element and avoid this?

Reply:
That's a good idea, makes things cleaner.